lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -22,6 +22,13 @@ class Auxiliary
     "-q" => [ false, "Run the module in quiet mode with no output"                         ]
   )
 
+  #
+  # Returns the hash of commands specific to auxiliary modules.
+  #
+  def action_commands
+      mod.actions.map { |action| [action.name, action.description] }.to_h
```

Comment:
Since actions are case insensitive and all existing commands are all lowercase, I'd like to see this updated to:

```suggestion
      mod.actions.map { |action| [action.name.downcase, action.description] }.to_h
```

I loaded up `auxiliary/admin/sap/cve_2020_6287_ws_add_user` (from a merged test branch) and get the following:

```
Auxiliary Commands
==================

    Command       Description
    -------       -----------
    ADD           Add the specified user
    REMOVE        Remove the specified user
    check         Check to see if a target is vulnerable
    exploit       This is an alias for the run command
    rcheck        Reloads the module and checks if the target is vulnerable
    recheck       This is an alias for the rcheck command
    rerun         Reloads and launches the auxiliary module
    rexploit      This is an alias for the rerun command
    run           Launches the auxiliary module
```

It would just be nice if the `REMOVE` and `ADD` actions were normalized to `remove` and `add` to fit in a little better.